.travis.php

```diff
@@ -19,8 +19,11 @@
     }
     echo "$dir\n";
 
-    $json = file_get_contents($dir.'/composer.json');
-    $package = json_decode($json);
+    $json = ltrim(file_get_contents($dir.'/composer.json'));
```

Comment:
You had that in the first version of your PR but removed it when switching to using Composer's validate command.